.ippen.media.txt

```diff
@@ -0,0 +1,58 @@
+# Author: HolgerAusB
+#
+# ippen.media is a German newspaper network that operates about 60+ websites
+# with (local) news and magazines, all or most of the newspaper sites with the same CMS.
+# magazine sites are not working yet.
+#
+#==========================
+#
+# If you host fultext-rss/wallabag on your own servers you may
+# a)
+# copy this file and name it for your domains e.g.:
+#        custom/hanauer.de.txt
+#        custom/ruhr24.de.txt
+#
+# or
+# b)
+# for fulltext-rss: add a fingerprint to the array in your custom_config.php:
+#
+# $options->fingerprints = array(
+#        //Ippen Media Newspaper Sites
+#        'var ippenErr = [], ippenPrevEH' => array('hostname'=>'fingerprint.ippen.media', 'head'=>true),
+#        // Posterous
+#        ...
+#        );
+#
+#==========================
+#
+# to get the source-feed add 'rssfeed.rdf' to the category-URL e.g.
+#    https://www.fr.de/hessen/rssfeed.rdf
+
+
+# strip author box and social media box
+# these boxes sometimes prevented main picure to show up
+strip_id_or_class: idjs-simpletab-nav-item
+strip_id_or_class: idjs-simpletab-content-close
+strip_id_or_class: id-AuthorList
+strip_id_or_class: id-Article-content-item-copyright
+
+# strip adverts
+strip_id_or_class: id-Article-content-item.id-Article-advert.id-Article-advert--ad3
+strip_id_or_class: id-Article-advert.id-Article-content-item
+strip_id_or_class: id-Article-content-item.id-Article-advert
+strip_id_or_class: id-Article-advert--ad3
+strip_id_or_class: id-Article-advert
+
+tidy: yes
+prune: yes
+
+test https://www.az-online.de/uelzen/stadt-uelzen/mehr-gaeste-in-der-uelzener-stadthalle-als-erlaubt-91282522.html
+test https://www.bgland24.de/bgland/region-berchtesgaden/bischofswiesen-ort28409/schoenau-am-koenigssee-wieder-grosses-interesse-bei-fit-durch-unser-gmoa-2022-91285772.html
+test https://www.bw24.de/baden-wuerttemberg/wolf-baden-wuerttemberg-sichtung-zollernalbkreis-spaziergaengerin-raubtier-nachweis-91297558.html
+test https://www.chiemgau24.de/chiemgau/chiemsee/gstadt-am-chiemsee-ort118608/gstadt-tourist-info-leiterin-berichtet-von-guter-auslastung-bei-gaestezahlen-trotz-corona-91288990.html
+test https://www.fnp.de/frankfurt/frankfurt-sachsenhausen-der-adlhochplatz-bekommt-einen-neuen-namen-91287869.html
+test https://www.fr.de/frankfurt/die-nfl-kommt-nach-frankfurt-91329620.html
+test https://www.hna.de/kassel/eigentuemer-sollen-sich-ruesten-91290102.html
+test https://www.kreis-anzeiger.de/lokales/wetteraukreis/mit-hoher-sicherheit-ein-wolf-91287739.html
+test https://www.kreiszeitung.de/lokales/bremen/bremen-wie-sich-osterholz-tenever-veraendert-hat-91304118.html
+test https://www.wetterauer-zeitung.de/wetterau/mahnwache-fuers-impfen-91318413.html
```

Comment:
```suggestion
test_url: https://www.az-online.de/uelzen/stadt-uelzen/mehr-gaeste-in-der-uelzener-stadthalle-als-erlaubt-91282522.html
test_url: https://www.bgland24.de/bgland/region-berchtesgaden/bischofswiesen-ort28409/schoenau-am-koenigssee-wieder-grosses-interesse-bei-fit-durch-unser-gmoa-2022-91285772.html
test_url: https://www.bw24.de/baden-wuerttemberg/wolf-baden-wuerttemberg-sichtung-zollernalbkreis-spaziergaengerin-raubtier-nachweis-91297558.html
test_url: https://www.chiemgau24.de/chiemgau/chiemsee/gstadt-am-chiemsee-ort118608/gstadt-tourist-info-leiterin-berichtet-von-guter-auslastung-bei-gaestezahlen-trotz-corona-91288990.html
test_url: https://www.fnp.de/frankfurt/frankfurt-sachsenhausen-der-adlhochplatz-bekommt-einen-neuen-namen-91287869.html
test_url: https://www.fr.de/frankfurt/die-nfl-kommt-nach-frankfurt-91329620.html
test_url: https://www.hna.de/kassel/eigentuemer-sollen-sich-ruesten-91290102.html
test_url: https://www.kreis-anzeiger.de/lokales/wetteraukreis/mit-hoher-sicherheit-ein-wolf-91287739.html
test_url: https://www.kreiszeitung.de/lokales/bremen/bremen-wie-sich-osterholz-tenever-veraendert-hat-91304118.html
test_url: https://www.wetterauer-zeitung.de/wetterau/mahnwache-fuers-impfen-91318413.html
```

I think it's `test_url` instead of just `test`